lib/kennel/tasks.rb

```diff
@@ -66,7 +70,7 @@ def git_push?
   # https://help.datadoghq.com/hc/en-us/requests/254114 for automatic validation
   desc "Verify that all used monitor  mentions are valid"
   task validate_mentions: :environment do
-    known = Kennel.send(:api)
+    known = Kennel::Api.new
```

Comment:
`Kennel::Tasks.kennel.api` ?
... seems a little inefficient that we create many different instances of the api

Reply:
I think the key word is "little". Show me that it makes a statistically meaningful difference and we discuss whether or not the breaking of encapsulation is worth it.

Reply:
prefer to not introduce bugs on purpose
"show me that it makes **no** difference" 🤷

Reply:
Calling a private method using `send` is a bug.

Reply:
it's not a "bug", but maybe bad api design :D
... can make it public 🤷

Reply:
Sure, we can.

Please could either approve or state that that is the change you're asking for? I'm having trouble finding this actionable.

Reply:
it's not a change I would have done, so wanted to check that there is a good reason for it
since you checked the call sites to make sure the caching is not broken we should be good to go